main/alpine-conf/s390x-bootloader.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/setup-disk.in b/setup-disk.in
+old mode 100644
+new mode 100755
+index efb7f41..c84c2eb
+--- a/setup-disk.in
++++ b/setup-disk.in
+@@ -459,6 +459,7 @@ install_mounted_root() {
+ 	case "$BOOTLOADER" in
+ 		grub) setup_grub "$mnt" "$root" "$modules" "$kernel_opts" "$bootdev" $disks ;;
+ 		syslinux) setup_syslinux "$mnt" "$root" "$modules" "$kernel_opts" "$bootdev" ;;
++		zipl) break;;
```

Comment:
We can do this because at https://github.com/alpinelinux/alpine-conf/blob/master/setup-disk.in#L485, installation of linux-vanilla (along with s390-tools) will trigger the bootloader, and it will create bootloader config, then run it.